doc/installation.rst

```diff
@@ -57,29 +56,3 @@ For beginners
 We provide here some installation tips for beginners in Python here:
 
 https://github.com/SpikeInterface/spikeinterface/tree/master/installation_tips
-
-
-
-Requirements
-------------
-
-:code:`spikeinterface.core` itself has only a few dependencies:
-
-  * numpy
-  * neo>=0.9.0
-  * joblib
-  * probeinterface
-  * tqdm
-
-Sub-modules have more dependencies, so you should also install:
-
-  * scipy
-  * h5py
-  * pandas
-  * sklearn
-  * matplotlib
-  * networkx
-  * datalad
-  * MEArec
-
-All sorters must installed independently.
```

Comment:
This is a doc, I think it is important to keep this. Even if this not totally up to date.
This is to give an overview on depencies.

Reply:
I'll put these back